component/Card/style.css

```diff
@@ -0,0 +1,196 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+body {
+  font-family:san-serif;
```

Comment:
Try creating variables for font-family, colors, etc. so that you do not have to specify it everywhere. You can create a general style file for body styles